libcaf_core/test/typed_spawn.cpp

```diff
@@ -372,6 +372,34 @@ CAF_TEST(check_signature) {
   run();
 }
 
+SCENARIO("state-classes may use typed pointers") {
+  GIVEN("a state class for a statically typed actor type") {
```

Comment:
Why is "state class" separated with a hyphen once and another time with a space?

Reply:
No reason, just a typo. Fixed. :)